get-latest-tgf.ps1

```diff
@@ -0,0 +1,41 @@
+$ErrorActionPreference = "Stop" #Make all errors terminating
+$env:PATH =$env:PATH+";."
```

Comment:
I think that does nothing. The current directory is already discoverable (unlike on Linux). If you type `tgf` and `tgf` is in the current directory, but not on the path, it will run.